lib/exception_hunter/error_creator.rb

```diff
@@ -6,9 +6,9 @@ def call(**error_attrs)
 
         ActiveRecord::Base.transaction do
           error_attrs = extract_user_data(error_attrs)
-          error = Error.new(error_attrs)
+          error = Error.new(error_attrs.except(:tags))
```

Comment:
I guess Code Climate was right on this one. Or kinda. `tags` should be something different from `error_attrs`, you can add an explicit keyword argument for it with a default. Plus I think it could just be tag since we are tagging with one label each time